gen/AArch64Arch_gen.ml

```diff
@@ -884,8 +902,11 @@ include
       let free_registers = allowed_for_symb
 
       type special = reg
+      type special2 = reg
```

Comment:
why do we need the type special2 here? As it is the same as special, can we not just use special